### PR TITLE
Ecpappserverjboss 719

### PR DESCRIPTION
**Pull request by eserbin**

### CODE REVIEW

File: specs/Services/CliCommandsGeneratorHelper.groovy [hunk 1]
```diff
@@ -138,6 +138,14 @@ class CliCommandsGeneratorHelper {
         return command
     }
 
+    static String shutDownHostDomain(String hostName) {
+        String command = "/host=$hostName:shutdown"
+        if(EnvPropertiesHelper.getVersion() =~ '6.0'){
```
Comment (ovasilko) [inline specs/Services/CliCommandsGeneratorHelper.groovy]: is this if statement needed here?

File: specs/StartServers.groovy [hunk 2]
```diff
@@ -561,6 +563,9 @@ class StartServers extends PluginTestHelper {
         then:
         assert runProcedureJob.getStatus() == 'error'
         assert runProcedureJob.getUpperStepSummary() =~ "Failed to connect to the controller"
+        
+        cleanup:
```
Comment (ovasilko) [inline specs/StartServers.groovy]: can it be easier to just provide incorrect controller ip and have the same errors?

File: specs/Services/CliCommandsGeneratorHelper.groovy (hunk 1 above)
Comment (eserbin) [reply specs/Services/CliCommandsGeneratorHelper.groovy]: fixed

File: specs/StartServers.groovy (hunk 2 above)
Comment (eserbin) [reply specs/StartServers.groovy]: as for me this scenario is more realistic

**Pull request merged**